reckoner/kube.py

```diff
@@ -91,7 +91,7 @@ def patch_metadata(self):
         """
         if self.overwrite:
             patch_metadata = self.metadata
-            logging.debug("Overwriting Namespace Metadata")
+            logging.info("Overwriting Namespace Metadata")
```

Comment:
Will this include the namespace that is having its metadata overwritten? If not, then I think we should add that info to the message

Reply:
Also, perhaps a debug or info message when we refuse to overwrite because it is false. That way the user isn't surprised when nothing happens. Is default for overwrite `false` ?

Reply:
Truncated output from the last changes. 

```
# reckoner plot course.yml -o a1 -o a2
2020-07-13 14:33:09 MacBook-Pro.local root[49384] INFO Installing a1
2020-07-13 14:33:09 MacBook-Pro.local root[49384] INFO Overwriting Namespace 'foobear' Metadata
2020-07-13 14:33:11 MacBook-Pro.local root[49384] INFO Installing a2
2020-07-13 14:33:11 MacBook-Pro.local root[49384] INFO Not Overwriting Metadata Label 'ManagedBy' in Namespace 'foobear'
```